src/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Horror hour</title>
+  </head>
+  <body>
+    <!--Header-->
+    <header>
+      <!--Navigation bar-->
+      <nav>
+        <ul id="nav-bar">
+          <li>
+            <a href="index.html" id="logo">Horror hour</a>
+          </li>
+          <li class="menu-option">
+            <a href="#" id="link_shows" class="links">Shows</a>
+          </li>
+          <li class="menu-option">
+            <a href="#" id="link_artists" class="links">Artists</a>
+          </li>
+          <li class="menu-option">
+            <a href="#" id="link_dates" class="links">Schedules</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--Main-->
+    <main>
+      <div id="items_section">
+        <div class="wrapper" id="items_container"></div>
+      </div>
+    </main>
+
+    <div class="commentSection">
+      <div class="modal"></div>
+    </div>
+
+    <div class="footer">
+      <p>
+        @ Created by
+        <a href="https://github.com/ijeomaodinko">Ijeoma Odinko</a>
+        and
+        <a href="https://github.com/VaneCode">Vanessa Bonito</a>
+        under Microverse program
+      </p>
+    </div>
```

Comment:
- I think It would be better to use HTML5 Semantic tags to follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 👌💯

> For Example
`<section class="commentSection">` instead of  `<div class="commentSection">`
and 
`<footer class="footer">` instead of `<div class="footer">`